### ISSUE TITLE
The limit of number of proposals returned 

### ISSUE THREAD

**Issue by mofokeng-chaka** [saltastroops/web-manager-2021#102]
The limit imposed on the backend for the maximum number of proposals returned is restricting users from seeing proposals later than semester 2015-2.

**Comment by hettlage** [comment]
Yes, but only if they try to view all (or at least too many) semesters at once. I don't think this is an issue, unless the user is not told about the fact there is a "problem". One solution would be to have the backend return an error if the maximum number of proposals is exceeded.

**Comment by mofokeng-chaka** [comment]
That is not always true, for instance, if a user selects semester 2006-1 from the options in the old web manager, 7 proposals are returned. However, this is not the case in the new web manager – nothing is returned because the first 1000 (backend limit) proposals are of semester 2015-2 and later. Maybe this is an issue that should be raised in the backend.
